manifests/install.pp

```diff
@@ -4,16 +4,31 @@
 #
 class consul::install {
 
+  case $::operatingsystem {
+    'windows': {
+      $binary_name = 'consul.exe'
+      $binary_mode = '0775'
```

Comment:
I just ran a test on the, and the binary_mode can be set to 755, but 555 does not work on windows 2016. I have pushed a version where I set it to 755 for both data_dir and binary